common/src/main/java/io/pravega/common/util/ConfigBuilder.java

```diff
@@ -78,6 +78,22 @@ private ConfigBuilder(String namespace, Properties properties, ConfigConstructor
         return this;
     }
 
+    /**
+     * Includes the given property and its value in the builder, if the provided value is not null.
+     *
+     * @param property The property to set.
+     * @param value    The value of the property. This must be of the same type as accepted by the Property.
+     * @param <V>      Type of the property.
+     * @return This instance.
+     */
+    public <V> ConfigBuilder<T> withDefaultIfNotExists(Property<V> property, V value) {
+        if (value != null) {
```

Comment:
This is not actually setting using the default, it is simply not adding anything at all, right? This doesn't look very good.

Reply:
Changed the approach to using `Optional`.